site/content/docs/4.3/layout/z-index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+title: Z-index
+description: While not a part of Bootstrap's grid system, z-indexes play an important part in how our components overlay and interact with one another.
+group: layout
+---
+
+Several Bootstrap components utilize `z-index`, the CSS property that helps control layout by providing a third axis to arrange content. We utilize a default z-index scale in Bootstrap that's been designed to properly layer navigation, tooltips and popovers, modals, and more.
+
+These higher values start at an arbitrary number, high and specific enough to ideally avoid conflicts. We need a standard set of these across our layered components—tooltips, popovers, navbars, dropdowns, modals—so we can be reasonably consistent in the behaviors. There's no reason we couldn't have used `100`+ or `500`+.
+
+We don't encourage customization of these individual values; should you change one, you likely need to change them all.
+
+{{< highlight scss >}}
+{{< zindex.inline >}}
+{{- $file := readFile "scss/_variables.scss" -}}
```

Comment:
I guess we could/should switch this to the `scss-docs` shortcode instead of an inline one? Unless you want to reduce the spaces after the variables, but it's your call.

Reply:
We should switch to remain consistent.